app/src/main/java/com/sakaimobile/development/sakaiclient20/ui/fragments/AnnouncementsFragment.java

```diff
@@ -113,10 +110,14 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         announcementRecycler.setLayoutManager(layoutManager);
         announcementRecycler.setItemAnimator(new DefaultItemAnimator());
 
+        announcementRecycler.setItemViewCacheSize(20);
```

Comment:
I assume this is needed because fast scrolling would drop frames if enough views aren't cached, so it would be nice if you could add a comment about that here (in case we ever come back to this). Or, if that's not the case, then explain what the large view cache is needed for.